### PR TITLE
Rename Or -> Xor per proposal in #189

### PR DESCRIPTION
**Pull request by mpilquist**

### CODE REVIEW

File: laws/src/main/scala/cats/laws/discipline/Arbitrary.scala [hunk 1]
```diff
@@ -14,8 +14,8 @@ object arbitrary {
   implicit def oneAndArbitrary[F[_], A](implicit A: Arbitrary[A], F: ArbitraryK[F]): Arbitrary[OneAnd[A, F]] =
     Arbitrary(F.synthesize[A].arbitrary.flatMap(fa => A.arbitrary.map(a => OneAnd(a, fa))))
 
-  implicit def orArbitrary[A, B](implicit A: Arbitrary[A], B: Arbitrary[B]): Arbitrary[A Or B] =
-    Arbitrary(Gen.oneOf(A.arbitrary.map(Or.left), B.arbitrary.map(Or.right)))
+  implicit def orArbitrary[A, B](implicit A: Arbitrary[A], B: Arbitrary[B]): Arbitrary[A Xor B] =
```
Comment (rossabaker) [inline laws/src/main/scala/cats/laws/discipline/Arbitrary.scala]: `xorArbitrary`

File: laws/src/main/scala/cats/laws/discipline/ArbitraryK.scala [hunk 2]
```diff
@@ -50,11 +50,11 @@ object ArbitraryK {
   implicit def constA[A](implicit A: Arbitrary[A]): ArbitraryK[Const[A, ?]] =
     new ArbitraryK[Const[A, ?]] { def synthesize[B: Arbitrary]: Arbitrary[Const[A, B]] = implicitly }
 
-  implicit def orA[A](implicit A: Arbitrary[A]): ArbitraryK[A Or ?] =
-    new ArbitraryK[A Or ?] { def synthesize[B: Arbitrary]: Arbitrary[A Or B] = implicitly }
+  implicit def orA[A](implicit A: Arbitrary[A]): ArbitraryK[A Xor ?] =
```
Comment (rossabaker) [inline laws/src/main/scala/cats/laws/discipline/ArbitraryK.scala]: `xorA`

File: laws/src/main/scala/cats/laws/discipline/ArbitraryK.scala [hunk 3]
```diff
@@ -50,11 +50,11 @@ object ArbitraryK {
...
+    new ArbitraryK[A Xor ?] { def synthesize[B: Arbitrary]: Arbitrary[A Xor B] = implicitly }
 
-  implicit def orB[B](implicit B: Arbitrary[B]): ArbitraryK[? Or B] =
-    new ArbitraryK[? Or B] { def synthesize[A: Arbitrary]: Arbitrary[A Or B] = implicitly }
+  implicit def orB[B](implicit B: Arbitrary[B]): ArbitraryK[? Xor B] =
```
Comment (rossabaker) [inline laws/src/main/scala/cats/laws/discipline/ArbitraryK.scala]: `xorB`

**Comment by mpilquist** [comment]
Thanks, fixed

**Comment by rossabaker** [comment]
:+1:

**Comment by non** [comment]
:+1:

**Pull request merged**